src/grammar.pegjs

```diff
@@ -633,20 +633,22 @@ regexp = '/' value:$(([^/\n] / '\\/')+)? closer:'/'? {
   };
 }
 
-quotedTerminal = '`' value:$(([^`\n] / ('\\`'))+)? closer:'`' {
+quotedTerminal = '`' value:$(([^`\n] / ('\\`'))+)? closer:'`' quantifier:('?')? {
```

Comment:
For completeness, this should include all of the possible quantifiers. See `nonTerminal` for an example of that